railib/api.py

```diff
@@ -172,17 +173,31 @@ def _get_collection(ctx, path: str, key=None, **kwargs):
 # Parses "multipart/form-data" responses. It returns the parts in a list.
 def _parse_multipart(content_type: str, content: bytes) -> list:
     result = []
+    content_type_json = b'application/json'
+    content_type_arrow = b'application/vnd.apache.arrow.stream'
     boundary = _extract_multipart_boundary(content_type)
+
     parts = content.split(b'\r\n' + boundary)
     for i, part in enumerate(parts):
         # fix the first part, it may contain the boundary
         if i == 0:
             part = part.split(boundary)[-1]
-        # skip the part which contains the following
+        # Skip "--\r\n"
         if b'--\r\n' in part:
             continue
-        # append the part
-        result.append(part)
+        # Part body and headers are separated with CRLFCRLF. Get the body.
+        part_value = part.split(b'\r\n\r\n')[-1]
```

Comment:
Is there not a library function for this already? Are we doing all the multipart parsing manually right now?

Reply:
I remember I tried to use requests_toolbelt but it was full of complexity, then @miazamrai picked it up while I was out and implemented it without it https://github.com/RelationalAI/rai-sdk-python/pull/59 maybe he can comment on this